service/worker/deployment/deployment_workflow.go

```diff
@@ -289,6 +289,9 @@ func (d *DeploymentWorkflowRunner) handleSyncState(ctx workflow.Context, args *d
 
 	// apply changes to "current"
 	if set := args.SetCurrent; set != nil {
+
+		d.logger.Info("Syncing from set-current")
```

Comment:
remove?

Reply:
not important though

Reply:
(this is the only change we made to service/worker/deployment directory, and it was made on accident)